ui/components/app/edit-gas-popover/edit-gas-popover.component.js

```diff
@@ -187,7 +189,14 @@ export default function EditGasPopover({
         );
         break;
       case EDIT_GAS_MODES.MODIFY_IN_PLACE:
-        dispatch(updateTransaction(updatedTxMeta));
+        newGasSettings.userEditedGasLimit = updatedTxMeta.userEditedGasLimit;
+        newGasSettings.userFeeLevel = updatedTxMeta.userFeeLevel;
+
+        dispatch(showLoadingIndication());
+        await dispatch(
+          updateTransactionGasFees(updatedTxMeta.id, newGasSettings),
+        );
+        dispatch(hideLoadingIndication());
```

Comment:
It will be nice to wrap this whole thing 
```
dispatch(showLoadingIndication());
        await dispatch(
          updateTransactionGasFees(updatedTxMeta.id, newGasSettings),
        );
        dispatch(hideLoadingIndication());
```
in hook so that all place in UI which use this function will not have to replicate the logic to show / hide loader.